packages/yarn-plugin/src/resolver/BackstageResolver.test.ts

```diff
@@ -107,25 +107,25 @@ describe('BackstageResolver', () => {
         ).toEqual(
           structUtils.makeDescriptor(
             structUtils.makeIdent('backstage', 'core'),
-            'backstage:1.23.45',
+            'backstage:^::v=1.23.45',
```

Comment:
The parameter name here is up for discussion - I figured using `v` to keep it brief was the best way to keep things readable, but happy to discuss alternatives!